packages/react-sdk/src/state/communication/discovery/matrixRtcSessionManagerImpl.ts

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2025 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StateEvent, WidgetApi } from '@matrix-widget-toolkit/api';
+import { clone, isEqual } from 'lodash';
+import { getLogger } from 'loglevel';
+import {
+  Observable,
+  Subject,
+  filter,
+  from,
+  interval,
+  switchMap,
+  takeUntil,
+} from 'rxjs';
+import {
+  RTCSessionEventContent,
+  STATE_EVENT_RTC_MEMBER,
+  isRTCSessionNotExpired,
+  newRTCSession,
+} from '../../../model';
+import {
+  DEFAULT_RTC_EXPIRE_DURATION,
+  isWhiteboardRTCSessionStateEvent,
+} from '../../../model/matrixRtcSessions';
+import { SessionState } from './sessionManagerImpl';
+import { Session, SessionManager } from './types';
+
+export class RTCSessionManagerImpl implements SessionManager {
```

Comment:
Suggest to rename it to `MatrixRtcSessionManagerImpl` to have consistent name with file and other code.